data-table-cell.html

```diff
@@ -21,6 +21,28 @@
       :host([hidden]) {
         display: none;
       }
+
+      :host([draggable]) {
```

Comment:
Rename `draggable` to `reorderable` and add it as a property in the `properties` of the new behavior